cmd/ebrelayer/main.go

```diff
@@ -0,0 +1,142 @@
+package main
+
+// -------------------------------------------------------------
+//      Main (ebrelayer)
+//
+//      Implements CLI commands for the Relayer service, such as
+//      initalization and event relay.
+// -------------------------------------------------------------
+
+import (
+	"encoding/hex"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client/rpc"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/ethereum/go-ethereum/common"
+	amino "github.com/tendermint/go-amino"
+	"github.com/tendermint/tendermint/libs/cli"
+
+	// "golang.org/x/crypto"
+
+	app "github.com/swishlabsco/cosmos-ethereum-bridge"
+	relayer "github.com/swishlabsco/cosmos-ethereum-bridge/cmd/ebrelayer/relayer"
+)
+
+const (
+	storeAcc       = "acc"
```

Comment:
this was already defined on a file above

Reply:
fixing in next push